airflow/config_templates/default_airflow.cfg

```diff
@@ -1027,6 +1027,10 @@ offset_field = offset
 use_ssl = False
 verify_certs = True
 
+[kubernetes_worker_resources]
```

Comment:
Is this required?

Reply:
The section `kubernetes_worker_resources` is not present in default_airflow.cfg  (in both 1.10.14 and 2.2.4) we have added this to populate resources field in k8s pod otherwise it sets the value to None